src/NServiceBus.Core/Persistence/Development/SagaPersister/DevelopmentSagaPersistence.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Runtime.Serialization.Json;
+    using NServiceBus.Sagas;
+
+    class DevelopmentSagaPersistence : Feature
+    {
+        internal DevelopmentSagaPersistence()
+        {
+            DependsOn<Sagas>();
+            Defaults(s => s.Set<ISagaIdGenerator>(new DevelopmentSagaIdGenerator()));
+            Defaults(s => s.SetDefault(StorageLocationKey, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, ".sagas")));
```

Comment:
@SimonCropp what do you think about making this backtrack looking for a .csproj and fallback to AppDomain.CurrentDomain.BaseDirectory if not found? (similar to dev transport looking for a .sln)